content/pages/irc.md

```diff
@@ -0,0 +1,66 @@
+Title: Using Internet Relay Chat (IRC)
+Category: pages
+Status: hidden
+Slug: irc
+
+The Women in Computer Science Undergraduate Committee uses a public [Internet
+Relay Chat](http://en.wikipedia.org/wiki/Internet_Relay_Chat) (IRC) channel
+to chat, plan events, and hold discussions. If you've never used IRC before,
+this document is designed to help you get set up.
+
+Our IRC channel, #wics, is located on the Open and Free Technology Community
+(OFTC) network. You can visit us by connecting to
+[irc.oftc.net](http://www.oftc.net/) or using our webchat, as detailed below.
+
+**Note:** All conversations in the channel are logged for future reference, but
+we do not publish our logs. Communication in the channel is governed by our
```

Comment:
We don't have any logging set up other than my personal logging. As soon as we get a server we can have a bot running.